src/main.cpp

```diff
@@ -823,6 +844,8 @@ QString UsageArguments ( char **argv )
         "                        supported for headless server mode)\n"
         "  -n, --nogui           disable GUI\n"
         "  -p, --port            set your local port number\n"
+        "  -Q, --qos             request quality of service in the IP header\n"
+        "                        (e.g. --qos 0) DF=0 AF31=104 CS4=128default EF=184\n"
```

Comment:
What about denoting the default like this?
```suggestion
        "                        common values: DF=0 AF31=104 CS4=128 EF=184\n"
        "                        default: --qos 128\n"
```

This makes it more verbose, of course, but I really had a hard time figuring out what "128default" is about, possibly because I do know QoS and TOS, but I certainly don't know the different classifications offhand. When reading "DF" in the network context, I initially thought about the "Don't fragment" bit...

Reply:
My intention was to show that you can set --qos to 0, and that 0 corresponds to DF, and that there are more values you can use, and that you don't need to set --qos to 128 because that is the default value.

Instead of 'common values:' I would say suggested values, because there are 256 possible values where most are not used (you need a table).
Showing "default: --qos 128" may suggest that you need to set the default.

The reason the command line is used; is for those that are determined to not use CS4 (value 128), otherwise, this function can be ignored.

It is difficult to explain things in two lines, however, there is no harm in experimenting and researching to discover what can be done. I can add extra documentation elsewhere.
Discussion at #1072 has all of the documentation, but maybe, it should be diststilled and put on the wiki. Windows users will have to follow the directions in a picture on #1072 because Windows overrides this function to DF=0 (best effort priority) for all internet traffic.

I would prefer to leave the Jamulus --help for qos the way they are, and put the rest on the wiki.